web/src/reducers/apd.js

```diff
@@ -369,23 +368,6 @@ const reducer = (state = initialState, action) => {
     case FETCH_ALL_APDS_FAILURE:
       return { ...state, fetching: false, error: action.error };
 
-    case SAVE_APD_SUCCESS:
-      return {
-        ...state,
-        byId: {
-          ...state.byId,
-          [action.data.id]: {
-            ...state.byId[action.data.id],
-            updated: getHumanTimestamp(action.data.updated)
-          }
-        },
-        data: {
-          ...state.data,
-          created: getHumanDatestamp(action.data.created),
-          updated: getHumanTimestamp(action.data.updated)
-        }
-      };
```

Comment:
I don't think we can get rid of this. Without it, the APD "last updated" time won't ever get updated. We use the APD state's `updated` property on the state dashboard (that's how we refer to the current landing page).